WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java

```diff
@@ -49,7 +49,7 @@ public static void refreshMetadata() {
         boolean isUserConnected = AccountHelper.isSignedIn();
```

Comment:
This is an aside since it's not tied to the current PR, but can't this routine be simplified to call the one above? Ex:
```
public static void refreshMetadata() {
    String username = AccountHelper.getDefaultAccount().getUserName();
    String email = AccountHelper.getDefaultAccount().getEmail();
    refreshMetadata(username, email);
}
```